src/WidgetIncluderModule.php

```diff
@@ -21,53 +21,64 @@ public function main(App $app)
 
     public function renderWidget(App $app, $widget_id, $options = [], $view = null)
     {
+
+        // get user context
         $user = $app->user();
-        
+
         /** @var Widget $widget */
         if (!$widget = Widget::where([
-            'id' => $widget_id
+            'id' => $widget_id,
```

Comment:
I noticed this after committing. Was not on purpose and done by my automatic formatter